core/src/test/java/org/fao/geonet/AbstractCoreIntegrationTest.java

```diff
@@ -299,6 +299,16 @@ public Element getSampleMetadataXml() throws IOException, JDOMException {
         return Xml.loadStream(resource.openStream());
     }
 
+    public Element getSampleISO19139MetadataXml() throws IOException, JDOMException {
+        final URL resource = AbstractCoreIntegrationTest.class.getResource("kernel/metadata.iso19139.xml");
```

Comment:
Maybe can replace both methods with a  generic method `getSampleMetadataXml(String resource)`?

Reply:
Added a `getSample` and keeping `getSamplISO*` for easier testing setup phase.